### PR TITLE
Add Room Page

### LINKED ISSUE #26: create add room page
add description

### PR DESCRIPTION
**Pull request by 235923**
closes #26

### CODE REVIEW

File: src/components/AddRoomForm/AddRoomForm.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                  mobile={12}
+                >
+                  <Field
+                    as={TextField}
+                    InputLabelProps={{ shrink: true }}
+                    className={styles.input}
+                    label={t('addRoom.closingTime')}
+                    type='date'
+                    name='closingTime'
+                    required
```
Comment (kaczor6418) [inline src/components/AddRoomForm/AddRoomForm.tsx:186]: This should be time no date
Screenshot from 2023-04-25 13-54-09

File: src/pages/AddRoomPage.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  {
+    id: 4,
+    name: 'Jan Kowalski4',
+  },
+  {
+    id: 5,
+    name: 'Jan Kowalski5',
+  },
+];
```
Comment (kaczor6418) [inline src/pages/AddRoomPage.tsx:52]: `USERS` should be fetched from backend

File: src/components/AddRoomForm/AddRoomForm.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        floor: '',
+        keyOwner: '',
+        maxCapacity: '',
+        openingTime: '',
+        roomNumber: '',
+        roomType: '',
+      }}
+      onSubmit={function () {
+        throw new Error('Function not implemented.');
+      }}
```
Comment (kaczor6418) [inline src/components/AddRoomForm/AddRoomForm.tsx:44]: this should send a request to backend

**Review by kaczor6418 — CHANGES_REQUESTED** [review]

File: src/components/AddRoomForm/AddRoomForm.module.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.input {
```
Comment (macszl) [inline src/components/AddRoomForm/AddRoomForm.module.scss:1]: We should be using styled components instead of scss

**Review by macszl — CHANGES_REQUESTED** [review]

**Comment by kaczor6418** [comment]
Closing as commits are fixed in #60

**Pull request closed without merging**